src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -127,23 +131,36 @@ public function getStyle($name)
     }
 
     /**
-     * {@inheritdoc}
+     * @return int
      */
-    public function format($message)
+    public function getDefaultWrapCutOption(): int
     {
-        return $this->formatAndWrap((string) $message, 0);
+        return $this->defaultWrapCutOption;
+    }
+
+    /**
+     * @param int $defaultWrapCutOption
+     *
+     * @return $this
+     *
+     * @see PrettyWordWrapperHelper
+     */
+    public function setDefaultWrapCutOption(int $defaultWrapCutOption)
```

Comment:
same, where do we use it? I tend to prefer having a `$cutOptions` as 3rd argument of `formatAndWrap()`, which makes it a clear new feature for 4.3